pipelinewise/fastsync/commons/tap_mysql.py

```diff
@@ -200,24 +200,24 @@ def get_table_columns(self, table_name):
                 SELECT column_name,
                     data_type,
                     column_type,
-                    CONCAT("REPLACE(", safe_sql_value, ", '\n', ' ')") safe_sql_value
+                    safe_sql_value
                 FROM (SELECT column_name,
                             data_type,
                             column_type,
                             CASE
                             WHEN data_type IN ('blob', 'tinyblob', 'mediumblob', 'longblob', 'geometry')
-                                    THEN concat('hex(`', column_name, '`)')
+                                    THEN CONCAT('REPLACE(hex(`', column_name, '`)', ", '\n', ' ')")
                             WHEN data_type IN ('binary', 'varbinary')
-                                    THEN concat('hex(trim(trailing CHAR(0x00) from `',COLUMN_NAME,'`))')
+                                    THEN concat('REPLACE(hex(trim(trailing CHAR(0x00) from `',COLUMN_NAME,'`))', ", '\n', ' ')")
                             WHEN data_type IN ('bit')
                                     THEN concat('cast(`', column_name, '` AS unsigned)')
                             WHEN data_type IN ('datetime', 'timestamp', 'date')
                                     THEN concat('nullif(`', column_name, '`,"0000-00-00 00:00:00")')
```

Comment:
This is somehow related to another recently reported issue of mysql 8 incompatibility.

What do you think can we use `str_to_date` as suggested in the link above and would that still work with mariadb/mysql5.x? And if yes, what do you think, can we add `str_to_date` into this PR or it should be addressed separately?

It'd be something like:
`nullif(`', column_name, '`, str_to_date("0000-00-00 00:00:00", "%Y-%m-%d %H:%i:%s"))


what do you think can we use

Reply:
Seems fine to me, killing two birds with one stone.
I ran few queries and this doesn't impact change the result.

Reply:
Will make a commit.

Reply:
I used the format `%Y-%m-%d %T`, `%T` is the equivalent of `%H:%i:%S`.